source/connect/tls-ssl-connection.txt

```diff
@@ -11,3 +11,106 @@ TLS/SSL Connection Tab
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`TLS / SSL` tab.
+
+      You can either use a ``Default`` connection or to set the TLS / SSL 
+      connection ``On`` or ``Off``. 
+
+      .. list-table::
+         :header-rows: 1 
+         :widths: 50 50
+      
+         * - Option
+           - Description
+
+         * - Default
+           - The ``Default`` option does not use the ``TLS / SSL`` connection option. 
+             When using the ``Default`` option, you can specify additional certificate 
+             options. To see more on the additional options available, see 
+             :ref:`<additional-tls>`.
+      
+         * - On
+           - The ``On`` option is enabled when using a :manual:`DNS seedlist (SRV) 
+             </reference/connection-string/#std-label-connections-dns-seedlist>` in 
+             the connection string. When TLS / SSL Connection is ``On``, you can 
+             specify additional certificate options for your connection string. 
+             To see more on the additional certificate options available, see 
+             :ref:`<additional-tls>`.
+
+             .. note::
+
+                If you try to connect a non-DNS seedlist connection string while 
+                using the ``On`` option, you will receive an error and be unable 
+                to connect. 
+
+         * - Off
+           - The ``Off`` option disables TLS / SSL for your connection. TLS / SSL 
+             is enabled by default when using a DNS seedlist (SRV) in a 
+             connection string. 
+
+      .. _additional-tls:
+
+      Additional TLS / SSL Options
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+      The ``Default`` and ``On`` options allow you to specify the following 
+      options:
+
+      .. list-table::
+         :header-rows: 1 
+         :widths: 50 50
+         
+         * - Option
+           - Description
+
+         * - Certificate Authority
+           - One or more certificate files from trusted Certificate Authorities 
+             to validate the certificate provided by the deployment. 
+
+         * - Client Certificate 
+           - Specifies the location of a local .pem file that contains either 
+             the client's TLS/SSL X.509 certificate or the client's TLS/SSL 
+             certificate and key.    
+
+         * - Client Key Password
+           - If the *Client Private Key* is protected with a password/passphrase, 
+             you must provide the password. 
+
+         * - tlsInsecure
+           - Disables various certificate validations.
+
+         * - tlsAllowInvalidHostnames
+           - Disables hostname validation of the certificate presented by 
+             the the deployment. 
+
+         * - tlsAllowInvalidCertificates
+           - Disable the validation of the server certificates.
+
+      .. warning:: 
+
+         Enabling these options may cause a security vulnerabilty. 
```

Comment:
I think this only applies to `tlsInsecure`, `tlsAllowInvalidHostnames`, and `tlsAllowInvalidCertificates`, so we should list those in this warning box.

Reply:
Done.